File: web/src/main/java/me/robinvdb/runetracker/database/DatabaseException.java
package me.robinvdb.runetracker.database;

public class DatabaseException extends RuntimeException {
    public DatabaseException() {
        super();
    }

    public DatabaseException(String message) {
        super(message);
    }

    public DatabaseException(String message, Throwable throwable) {
        super(message, throwable);
    }

    public DatabaseException(Throwable throwable) {
        super(throwable);
    }
}

File: web/src/main/java/me/robinvdb/runetracker/database/DatabaseException.kt
package me.robinvdb.runetracker.database

class DatabaseException : RuntimeException {
    constructor() : super() {}

    constructor(message: String) : super(message) {}

    constructor(message: String, throwable: Throwable) : super(message, throwable) {}

    constructor(throwable: Throwable) : super(throwable) {}
}
